app/classifier/annotation-renderer/svg.jsx

```diff
@@ -86,7 +87,7 @@ export default class SVGRenderer extends React.Component {
 
     if (this.props.annotation.task) {
       taskDescription = this.props.workflow.tasks[this.props.annotation.task];
-      ({ InsideSubject } = tasks[taskDescription.type]);
+      (({ InsideSubject } = tasks[taskDescription.type]));
```

Comment:
Any idea why the extra brackets were added here?

Reply:
Haven't the foggiest. Only think I can think of is an auto-linter I had enabled going a bit funny. Will commit to remove them but will hold off in case anything else is spotted

Reply:
They don't cause any problems, as far as I can tell. Just a bit weird.

Reply:
Interesting that it's also stripping newlines at the end of files. That's not behaviour I'd anticipated